readthedocs/templates/projects/project_advanced_settings_helptext.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+<p class="empty">
```

Comment:
Used the same patter that we have in settings https://readthedocs.org/dashboard/docs/edit/